src/client/package.json

```diff
@@ -22,7 +22,8 @@
   "scripts": {
     "clean": "rm -rf dist",
     "start": "npm run dev",
-    "lint": "eslint src",
+    "dev:local": "npm run dev -- --endpoint local",
```

Comment:
what do you mean config we pass? Do you mean a file on disk, like `environment-config.json`?

Ben did suggest this, we thought of just hardcoding it with a comment. I guess we could have a single file with 1 prop in there for url.